src/ItkVtkViewProxy.js

```diff
@@ -144,9 +140,9 @@ function ItkVtkViewProxy(publicAPI, model) {
   model.interactor.onMouseMove((event) => {
     updateAnnotations(event);
   });
-  model.interactor.onMouseWheel((event) => {
-    updateAnnotations(event);
-  });
+  // model.interactor.onMouseWheel((event) => {
+  //  updateAnnotations(event);
+  // });
```

Comment:
the mouse wheel event that is forwarded does not have information regarding position. If you want to reuse a position information from a previous event (mousemove) you'll need to explicitly store it.

Reply:
Thanks for the note. Removed.